controllers/activitybuffers.js

```diff
@@ -26,7 +27,7 @@ activityBufferRouter.post('/activities', async (req, res) => {
     return res.status(500).send(addedActivity.error)
   }
 
-  res.status(201).json(addedActivity)
+  res.status(201).json(await activityService.prepareActivity(addedActivity))
```

Comment:
Maybe change so these prepare functions don't have to be called from controllers separately?